### PR TITLE
db refactor: Replace '' with NULL in all 3 string-type columns in postgres

### LINKED ISSUE #3801: Refactor db fields containing url to remove invalid url data
There are some fields in the existing open-event database in which invalid strings like `' '` or any other are being stored. Due to this when using field.Url() in the api, an error is raised 
```
list indices must be integer, not str
```
Goals:
provided data migration to fix this

### PR DESCRIPTION
**Pull request by shubham-padia**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Replace `''` with NULL in all 3 stringType columns in postgres

#### Changes proposed in this pull request:

- Replace `''` with NULL in all 3 stringType columns in postgres
- make few columns nullable as they previously contained `''` 

Fixes #3801

### CODE REVIEW

**Comment by shubham-padia** [comment]
> @shubham-padia this issue is there in a few speaker columns too. And code of conduct

@niranjan94 Are you talking about making the columns nullable here. If not, the `''` is checked across all tables of the db. Also code of conduct is a column of event, so it is covered when checking the event table.

File: migrations/versions/1b9c8698c632_.py [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+               execution_options=None)
+    op.execute("SELECT replace_in_table('access_codes', '''''', 'NULL', true)",
+               execution_options=None)
+    op.execute("SELECT replace_in_table('user_system_role', '''''', 'NULL', true)",
+               execution_options=None)
+    op.execute("SELECT replace_in_table('message_settings', '''''', 'NULL', true)",
+               execution_options=None)
+    op.execute("SELECT replace_in_table('image_sizes', '''''', 'NULL', true)",
+               execution_options=None)
+    op.execute("SELECT replace_in_table('speaker', '''''', 'NULL', true)",
```
Comment (shubham-padia) [inline migrations/versions/1b9c8698c632_.py]: Here is the speaker Table :)

**Comment by shubham-padia** [comment]
@niranjan94 ^^ :)

**Comment by niranjan94** [comment]
Let this be merged after one more review+approval.

@fossasia/open-event-server-devs

**Pull request merged**